fs/dafsclient/rmtclient.cpp

```diff
@@ -727,13 +727,16 @@ void CRemoteBase::connectSocket(SocketEndpoint &ep, unsigned connectTimeoutMs, u
             //PrintStackReport();
         }
         bool ok = true;
+        unsigned connecttimeoutMs = DEFAULT_CONNECT_TIME;
         try
         {
+            CCycleTimer timer;
```

Comment:
previously the timer was outside of the loop, i.e. 'connectTimeoutMs' was for the whole attempt time, now timer is (in effect) reset each attempt. Should move up outside of while loop.